src/molgenis/capice/main_train.py

```diff
@@ -242,6 +241,7 @@ def train(self, test_set: pd.DataFrame, train_set: pd.DataFrame):
             verbosity=verbosity,
             objective='binary:logistic',
             booster='gbtree',
+            tree_method='exact',
```

Comment:
After conversation: for model consistency setting tree method to arguably the better `exact` is beneficial for future releases.

Reply:
Currently uses default of `auto`, though in practice this currently results in `exact` ([source](https://github.com/dmlc/xgboost/blob/v1.7.4/src/gbm/gbtree.cc#L139) in comparison to our current train dataset size). In case our dataset would outgrow this threshold, suddenly using a different algorithm doesn't seem like preferred behaviour.